index.html

```diff
@@ -0,0 +1,186 @@
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Three -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Four -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card five -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Six -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="about">
+          <h4 id="about-me">About me</h4>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hestiate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
```

Comment:
- [ ] It seems you forgot to add some about me sections which are required in project requirements. You should try to stick to Figma design as much as possible to match the design as you will be doing in your future job after microverse where you will be required to implement Figma mockups into real projects strictly. 

your design:

![image](https://user-images.githubusercontent.com/90524466/170773334-c8a0e0e6-01ad-42fb-b84f-e6f4a3475a7f.png)

Figma design(forgotten sections):


![image](https://user-images.githubusercontent.com/90524466/170774027-8eed20a9-e97a-410a-aa0c-053b5a71370a.png)